dev-qt/qtgui/qtgui-5.10.9999.ebuild

```diff
@@ -71,6 +71,8 @@ PDEPEND="
 	ibus? ( app-i18n/ibus )
 "
 
+PATCHES=( "${FILESDIR}/${PN}-5.9.4-opengl.patch" ) # QTBUG-66420
```

Comment:
IIRC upstream closed the 5.10 branch (no more releases from it), so we should just kill 5.10.9999 instead of wasting time backporting fixes to it.